CMakeLists.txt

```diff
@@ -69,6 +71,10 @@ add_library(GraphiT OBJECT
     "${CMAKE_CURRENT_SOURCE_DIR}/external/graphi-t/src/gft/util.cpp")
 target_include_directories(GraphiT PUBLIC "${CMAKE_CURRENT_SOURCE_DIR}/external/graphi-t/include")
 
+if (NOT ANDROID)
+  # Backward-cpp
+  add_subdirectory("${CMAKE_CURRENT_SOURCE_DIR}/external/backward-cpp")
```

Comment:
Doesn't work on Android? Or is it not yet tested?

Reply:
it doesn't work on android since backward-cpp depends on some linux specific headers :(